code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -441,6 +462,7 @@
 /datum/tarot/strength
 	name = "XI - Strength"
 	desc = "May your power bring rage."
+	extended_desc = "grants the user strength only matched by powerful vampires. The strength blocks use of ranged weapons."
```

Comment:
```suggestion
	extended_desc = "grants the user strength only matched by powerful vampires. The strength renders the user unable to handle ranged weapons."
```

I think "blocks use of ranged weapons" could be misunderstood as "nobody can use ranged weapons on this guy"